documentation/adoc/cluster-controller.adoc

```diff
@@ -247,12 +247,12 @@ The configurable fields are the following :
 * `namespace`: the Kubernetes namespace (OpenShift project) in which the topic controller watches for topic ConfigMaps.
 Default is the namespace where the topic controller is running
 * `reconciliationIntervalMs`: the interval between periodic reconciliations. Default is "15 minutes"
-* `zookeeperSessionTimeout`: the Zookeeper session timeout. Default is "20 seconds"
+* `zookeeperSessionTimeoutMs`: the Zookeeper session timeout. Default is "20 seconds"
```

Comment:
Do we want to list the defaults in milliseconds when the value is in milliseconds?